### DISCUSSION TITLE
Tygo is not finding config for my package

### DISCUSSION THREAD

**Discussion by ManoloTonto1** [gzuidhof/tygo#14] [Q&A] [answered]
Hello there, I just installed tygo and wanted to start using it but, i can't get it to pickup my current package directory. 
my config file looks like this.
![image](https://user-images.githubusercontent.com/74105210/215362276-043c896e-0a63-4eeb-a585-1540b77ff130.png)

and the error i am getting:
`2023/01/30 00:27:05 Config not found for package grapeseed/models`
I am running this on go version 1.19  and at the root of my project.

Thanks for the help!!

**Comment by gzuidhof** [comment, marked as answer]
Hi Manuel,

As the comment suggests, try to define it as you would import it, so
generally it would be something like “github.com/my/repo/models”, not as a
relative path

**Reply by ManoloTonto1** [reply]
Amazing! works like a charm! 
Thank you so much for your help!

**Reply by vlauffer** [reply]
Sorry if this is a stupid question, but does this mean that the tool won't work unless the models file isn't already on github?
